layers/+email/mu4e/packages.el

```diff
@@ -38,4 +39,10 @@
 
       (when mu4e-account-alist
         (add-hook 'mu4e-compose-pre-hook 'mu4e/set-account)
-        (add-hook 'message-sent-hook 'mu4e/mail-account-reset)))))
+        (add-hook 'message-sent-hook 'mu4e/mail-account-reset))
+      (if mu4e-use-maildirs
```

Comment:
It's cosmetic, but a `when` would be more appropriate and consistent with the rest of the code.